CMakeLists.txt

```diff
@@ -37,35 +37,35 @@ ecbuild_bundle_initialize()
 # --------------------------
 
 # jedi-cmake
-ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda/jedi-cmake.git" TAG 1.0.0 )
+ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda/jedi-cmake.git" BRANCH develop UPDATE )
 
 # ECMWF libraries
 find_package( eckit 1.11.6 QUIET)
 if (${eckit_FOUND})
   message(STATUS "Found ECKIT package, not building as part of the bundle" )
 else()
-  ecbuild_bundle( PROJECT eckit GIT "https://github.com/jcsda-internal/eckit.git" TAG 1.11.jcsda )
+  ecbuild_bundle( PROJECT eckit GIT "https://github.com/jcsda-internal/eckit.git" BRANCH release-stable UPDATE )
```

Comment:
I was thinking it probably wouldn't be long before develop doesn't build with the tagged versions of these ecmwf repos. But this is develop so it would be easy to change that if we need to.